cli/printer.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const Printer = {
+  /**
+   * @param {!{info: !Function}} log
+   * @param {string=} url
+   * @param {!{log: !Function}} output
+   * @param {!Array<?>} results
+   */
+  json: function(log, output, url, results) {
+    log.info('\n\n\nJSON Lighthouse results:', url ? url : '');
```

Comment:
since this is the CI context and the default URL right now is defined inside of the runner and not at the CI level the knowledge of the URL (and the default URL) is at the wrong level. The results do not currently include the URL in the report (they should)

I aimed to explicitly define the dependencies  here so that we could have these kind of revelations and conversations... so :+1: it is working!